services/storage-users/pkg/config/config.go

```diff
@@ -203,6 +203,7 @@ type PosixDriver struct {
 	UseSpaceGroups bool `yaml:"use_space_groups" env:"STORAGE_USERS_POSIX_USE_SPACE_GROUPS" desc:"Use space groups to manage permissions on spaces." introductionVersion:"6.0.0"`
 
 	WatchType               string `yaml:"watch_type" env:"STORAGE_USERS_POSIX_WATCH_TYPE" desc:"Type of the watcher to use for getting notified about changes to the filesystem. Currently available options are 'inotifywait' (default), 'gpfswatchfolder' and 'gpfsfileauditlogging'." introductionVersion:"6.0.0"`
+	WatchMountPoint         string `yaml:"watch_mount_point" env:"STORAGE_USERS_POSIX_WATCH_MOUNT_POINT" desc:"The local path where the posixfs file system is mounted. Paths from events will be considered relative to this path." introductionVersion:"6.0.0"`
```

Comment:
`introductionVersion:"6.0.0"` --> This is definitely wrong !!
Either it is `7.0.0` if it goes into v7 or it must be `%%NEXT%%` to be marked added for the next release.